hirefire/procs/celery.py

```diff
@@ -1,22 +1,31 @@
 from __future__ import absolute_import
 from collections import Counter
 from itertools import chain
+from logging import getLogger
 
 from celery.app import app_or_default
 
 from ..utils import KeyDefaultDict
 from . import Proc
 
 
+logger = getLogger(__name__)
```

Comment:
That makes sense. It's a small enough project, so it would allow more flexibility when refactoring.